packages/components/src/components/typography/Link/index.tsx

```diff
@@ -56,7 +56,17 @@ interface Props {
 }
 
 const Link = ({ icon, iconProps, ...props }: Props) => (
-    <A href={props.href} target={props.target || '_blank'} rel="noreferrer noopener" {...props}>
+    <A
+        href={props.href}
+        target={props.target || '_blank'}
+        rel="noreferrer noopener"
+        {...props}
```

Comment:
hmm I think if someone actually add custom `onClick` prop to their Link it will be passed here by spreading `props` and line below will simply override it and original onClick handler that dev passed will get lost.
So just to be sure if someone actually did it, we could call passed `props.onClick` func in our custom onClick handler below (before e.stopPropagation). But please double check if I am correct or just talking bullshit 😁